bin/install.sh

```diff
@@ -341,6 +346,20 @@ echo WEB_CONCURRENCY=$webConcurrency >> $envFile
 
 echo -e "\n\nGreat! That's all we need, we will have your Waffle.io Takeout ready shortly."
 
+######################################################
+# Store off environment configuration in /etc/waffle #
+######################################################
+echo -e "\n\nSaving environment configuration in /etc/waffle/"
+sudo mkdir -p /etc/waffle/ca-certificates
+sudo cp $envFile /etc/waffle/environment.list
```

Comment:
wondering what the best solution is... now we have 3 copies, and we overwrite this file on each install. Should we check for existence first? Should we copy sooner, and only update the one in /etc/waffle?